packages/table/preview/index.tsx

```diff
@@ -417,7 +443,34 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
             this.toTruncatedPopoverModeLabel,
             this.handleNumberStateChange,
             "enableCellTruncation",
-            true,
+            [true],
+        );
+
+        // Table: "Scroll to"
+        const scrollToRegionTypeSelectMenu = this.renderSelectMenu(
```

Comment:
This whole area is a mess now. Wanted to get a preview up quickly, but would like to clean this up before merging.